x-pack/plugins/timelines/public/components/t_grid/standalone/index.tsx

```diff
@@ -389,7 +388,6 @@ const TGridStandaloneComponent: React.FC<TGridStandaloneProps> = ({
                       itemsPerPage: itemsPerPageStore,
                     })}
                     totalItems={totalCountMinusDeleted}
-                    indexNames={indexNames}
```

Comment:
I think we still want to pass this in. Why remove it?

Reply:
The `indexNames` prop is used in this component to `tGridActions.createTGrid`, but inside the body now I use the `_index` in the data response. This `indexNames` value was not working anymore in the request to update the status of the items, it is not used anymore in the body component, so I removed the prop.